samples/sensor/hts221/prj.conf

```diff
@@ -1,5 +1,7 @@
 CONFIG_STDOUT_CONSOLE=y
 CONFIG_I2C=y
 CONFIG_SENSOR=y
-CONFIG_HTS221_TRIGGER_NONE=y
+#Default trigger value:
+#CONFIG_HTS221_TRIGGER_NONE=y
+
```

Comment:
Why are you adding commented out code? This should be removed.

Reply:
Sure